.github/workflows/vast.yaml

```diff
@@ -990,6 +990,19 @@ jobs:
           echo "::set-output name=tar_gz::${TAR_GZ}"
           echo "::set-output name=deb::${DEB}"
 
+      - name: Run Integration Tests
+        run: |
+          tar xf ./result/${{ steps.get_artifact_names.outputs.tar_gz }}
+          nix shell --command python vast/integration/integration.py --app opt/vast/bin/vast
+
+      - name: Upload Integration Test Logs on Failure
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: "vast-integration-test-nix-static-${{ matrix.build.compiler }}-${{ matrix.configure.tag }}"
+          path: "${{ env.BUILD_DIR }}/vast/vast-integration-test"
```

Comment:
I don't see $BUILD_DIR set in the environment. What you want here is the directory that CMake does the build in, i.e., the directory that is passed to CMake via `-B path/to/build`, and then inside that `/vast/vast-integration/test`.